.maintain/gitlab/check_polkadot_companion_status.sh

```diff
@@ -85,11 +88,16 @@ else
 fi
 
 curl -H "${github_header}" -sS -o companion_pr_reviews.json \
-  ${github_api_polkadot_pull_url}/${pr_companion}/reviews 
+  ${github_api_polkadot_pull_url}/${pr_companion}/reviews
 
-if [ -n "$(jq -r -e '.[].state | select(. == "CHANGES_REQUESTED")' < companion_pr_reviews.json)" ] && \
-  [ -z "$(jq -r -e '.[].state | select(. == "APPROVED")' < companion_pr_reviews.json)" ]
-then
+# If there are any 'CHANGES_REQUESTED' reviews for the *current* review
+if [ "$(jq -r -e '.[] | select(.state == "CHANGES_REQUESTED").commit_id' < companion_pr_reviews.json)" = "$pr_head_sha" ]; then
```

Comment:
haven't tested myself, but is there a chance that more than the head of a pr is in this list?

Reply:
Hmm I think it will break for changes with more than one `CHANGES_REQUESTED` review. Good catch.